internal/driver/identityserver.go

```diff
@@ -50,22 +51,36 @@ func NewIdentityServer(ctx context.Context, linodeDriver *LinodeDriver) (*Identi
 	return identityServer, nil
 }
 
-// GetPluginInfo(context.Context, *GetPluginInfoRequest) (*GetPluginInfoResponse, error)
+// GetPluginInfo returns information about the CSI plugin.
+// This method is REQUIRED for the Identity service as per the CSI spec.
+// It returns the name and version of the CSI plugin.
 func (linodeIdentity *IdentityServer) GetPluginInfo(ctx context.Context, req *csi.GetPluginInfoRequest) (*csi.GetPluginInfoResponse, error) {
-	klog.V(5).Infof("Using default GetPluginInfo")
+	log, _, done := logger.GetLogger(ctx).WithMethod("GetPluginInfo")
+	defer done()
+
+	log.V(2).Info("Processing request")
 
 	if linodeIdentity.driver.name == "" {
 		return nil, status.Error(codes.Unavailable, "Driver name not configured")
 	}
 
+	log.V(2).Info("Successfully completed")
```

Comment:
logging on line 61 and 67 doesnt seem right to me... we havent really processed the request yet... it happens when we return with no error on line 68-71.
I would suggest removing these and adding those logs before and after the function/method that calls `GetPluginInfo`

Reply:
Removed it. I had that there to sort of capture entry and exit for the func. But these are super simple grpc funcs() so we should be fine just removing `Successfully completed `